src/main/java/edu/hawaii/its/groupings/configuration/AppConfig.java

```diff
@@ -2,9 +2,18 @@
 
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
 
 @Configuration
 @ComponentScan(basePackages = "edu.hawaii.its")
+@PropertySources({
+        @PropertySource("classpath:custom.properties"),
+        @PropertySource(value = "file:${user.home}/.${user.name}-conf/uh-groupings-ui-overrides.properties",
+                ignoreResourceNotFound = true),
+        @PropertySource(value = "file:${user.home}/.${user.name}-conf/myiam-overrides.properties",
+                ignoreResourceNotFound = true)
```

Comment:
Since we renamed the myiam-overrides properties to uh-groupings-ui-overrides, we can delete the highlighted lines.